ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -49,7 +50,8 @@ SettingsContentBase {
         Communities = 1,
         Accounts = 2,
         Collectibles = 3,
-        Web = 4
+        //Assets = 4,
+        Web = 5
```

Comment:
That's not possible, it must be `= 4` bc it's mainly used to refer the stack layout current index.